scripts/core/superdesk-api.d.ts

```diff
@@ -757,6 +757,10 @@ declare module 'superdesk-api' {
                     ruleHandlers?: {[key: string]: IIngestRuleHandlerExtension};
                 };
             };
+            aiAssistant?: {
```

Comment:
superdesk-api should be limited to core functionalities. AI widget is an add-on so it shouldn't go here.

Instead you should make it a regular extension and expose customization options from the extension itself. You can check how it's done with [broadcasting](https://github.com/superdesk/superdesk-insajder/blob/0b95804d6af160608b4156d0f092af579dc9e2ab/client/index.ts#L9)

Reply:
> Can you suggest me a good way to handle the id of the extension? I'm using that to track a bunch of things and I need it to be accessible from belga's repo and also client-core.

after you make it a regular extension, you won't have this problem either.